bgpd/bgp_mplsvpn.c

```diff
@@ -27,9 +27,12 @@
 #include "stream.h"
 #include "queue.h"
 #include "filter.h"
+#include "mpls.h"
 #include "lib/json.h"
+#include "lib/zclient.h"
```

Comment:
You do not need `lib` here, the include path is specified by the compiler options. If you wish to remove the `lib/` from `lib/json.h` on the line above that would be a welcome change as well.